packages/codemod/src/utils/prop-utils.ts

```diff
@@ -1,31 +1,129 @@
-import { ASTPath, JSCodeshift, JSXIdentifier, JSXOpeningElement } from "jscodeshift";
+import {
+  ASTPath,
+  Collection,
+  JSCodeshift,
+  JSXAttribute,
+  JSXElement,
+  JSXIdentifier,
+  JSXOpeningElement,
+  Literal
+} from "jscodeshift";
+import { logPropMigrationError } from "./report-utils";
 
 /**
  * Updates a prop name in a JSX element
  */
 export function updatePropName(
-  elementPath: ASTPath<JSXOpeningElement>,
+  j: JSCodeshift,
+  elementPath: ASTPath<JSXElement>,
   propsNamesMappingOldToNew: Record<string, string>
 ): void {
-  const attributes = elementPath.node?.attributes;
-  if (!attributes) return;
-
-  attributes.forEach(attr => {
-    if (
-      attr.type === "JSXAttribute" &&
-      attr.name.type === "JSXIdentifier" &&
-      propsNamesMappingOldToNew[attr.name.name]
-    ) {
-      attr.name.name = propsNamesMappingOldToNew[attr.name.name];
-    }
-  });
+  j(elementPath)
+    .find(JSXOpeningElement)
+    .find(JSXIdentifier)
+    .forEach(identifierPath => {
+      if (propsNamesMappingOldToNew[identifierPath.node.name]) {
+        identifierPath.node.name = propsNamesMappingOldToNew[identifierPath.node.name];
+      }
+    });
 }
 
 /**
  * Checks for whether a prop is used in a JSX element
  */
-export function isPropExists(j: JSCodeshift, elementPath: ASTPath<JSXOpeningElement>, propName: string): boolean {
-  const attributes = elementPath.node?.attributes;
-  if (!attributes) return false;
-  return j(elementPath).find(JSXIdentifier, { name: propName }).size() > 0;
+export function isPropExists(j: JSCodeshift, elementPath: ASTPath<JSXElement>, propName: string): boolean {
+  return j(elementPath).find(JSXOpeningElement).find(JSXIdentifier, { name: propName }).size() > 0;
+}
+
+/**
+ * Finds props on a JSX element
+ */
+export function findProps(j: JSCodeshift, elementPath: ASTPath<JSXElement>, ...propNames: string[]) {
+  return j(elementPath)
+    .find(JSXOpeningElement)
+    .find(JSXAttribute)
+    .filter(attr => {
+      const attrName = attr.node.name.type === "JSXIdentifier" ? attr.node.name.name : "";
+      return propNames.includes(attrName);
+    });
+}
+
+/**
+ * Removes a prop from a JSX element
+ */
+export function removeProp(j: JSCodeshift, elementPath: ASTPath<JSXElement>, ...propsNames: string[]): void {
+  j(elementPath)
+    .find(JSXOpeningElement)
+    .find(JSXAttribute)
+    .forEach(attrPath => {
+      if (attrPath.node.name.type === "JSXIdentifier" && propsNames.includes(attrPath.node.name.name)) {
+        j(attrPath).remove();
+      }
+    });
+}
+
+/**
+ * Gets the computed value of a prop
+ */
+function getPropValue(j: JSCodeshift, prop: JSXAttribute): undefined | boolean | Literal["value"] | string {
+  const value = prop.value;
+
+  // boolean flag value (e.g. <Button disabled />)
+  if (value === null) {
```

Comment:
it would get into the
```if (value?.type === "JSXExpressionContainer") {```
`null` only means that it is a boolean `true` flag prop (without "" or {}, declaring `null` must be inside {})